access/slackbot/bot.go

```diff
@@ -103,6 +103,10 @@ func (b *Bot) Respond(ctx context.Context, reqID string, reqData requestData, st
 		return trace.Wrap(err, "failed to send update: %v", err)
 	}
 
+	if err := rsp.Body.Close(); err != nil {
+		return trace.Wrap(err, "failed to close body: %v", err)
+	}
```

Comment:
this should be in a defer, otherwise `ioutil.ReadAll` below will fail.
also, consider adding a unit test for this